### ISSUE TITLE
Turn off printing for each sampler

### ISSUE THREAD

**Issue by itsdfish** [StatisticalRethinkingJulia/MCMCBenchmarks.jl#2]
Between now and the time we finish the models, we should turn off printing so it does not invalidate the results. Turing doesn't suppress printing on master, but it might on the official release.

**Comment by goedman** [comment]
I believe I was never able to completely suppress the printing, but it worked in notebooks to not show intermediate results.

**Comment by itsdfish** [comment]
That might suffice. Although it might be nice to have the option to suppress printing from the REPL or Juno. I'm not sure if that is worth filing an issue for a feature request.

**Comment by itsdfish** [comment]
It looks like DynamicHMC does not have the option to turn off printing. I'll go ahead and close this.

**Comment by goedman** [comment]
It does:
```
  chain, NUTS_tuned = NUTS_init_tune_mcmc(∇P, Nsamples, report=ReportSilent());
```

**Comment by itsdfish** [comment]
Great! I'll add that once everything is merged on your end.

**Comment by itsdfish** [comment]
Done.
